docs-app/package.json

```
@@ -21,7 +21,7 @@
     "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
     "lint:js:fix": "eslint . --fix",
     "lint:types": "glint",
-    "start": "ember serve",
+    "start": "concurrently 'ember serve' 'pnpm sync-pnpm --watch' --names 'docs-app serve,docs-app sync built package'",
```

a space here would make it easier to read, but maybe that will stop it working

```suggestion
    "start": "concurrently 'ember serve' 'pnpm sync-pnpm --watch' --names 'docs-app serve, docs-app sync built package'",
```

---

Yeah - the example in the docs didn't have a space so I went without but I can experiment and see...